cmd/common.go

```diff
@@ -26,6 +26,19 @@ var (
 	assumeYes  bool
 )
 
+func performValidate(ctx context.Context, kongClient *kong.Client, entity interface{}, entityType string) (bool, error) {
+	endpoint := fmt.Sprintf("/schemas/%s/validate", entityType)
+	req, err := kongClient.NewRequest("POST", endpoint, nil, entity)
+	if err != nil {
+		return false, err
+	}
+	resp, err := kongClient.Do(ctx, req, nil)
+	if err != nil {
+		return false, err
```

Comment:
Please wrap this error with some contextual info using fmt.Errorf.
fmt.Errorf("validate entity '%s': %s",entittyType,err)

Reply:
Since there are two places where we may return an error here, I was wrapping it with some context on the caller side:

```
_, err := performValidate(ctx, kongClient, entity, entityType)
...
output = color.New(color.FgRed, color.Bold).Sprintf("%s: %v", entityType, err)
```

If you prefer to have it here though, that's fine to me!